go.mod

```diff
@@ -9,45 +9,34 @@ require (
 	github.com/Masterminds/goutils v1.1.0 // indirect
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/Microsoft/hcsshim v0.8.10 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef // indirect
 	github.com/bmatcuk/doublestar v1.3.2
 	github.com/bndr/gojenkins v1.0.1
-	github.com/containerd/containerd v1.4.1 // indirect
-	github.com/docker/docker v1.4.2-0.20200114201811-16a3519d870b // indirect
 	github.com/elliotchance/orderedmap v1.3.0
 	github.com/evanphx/json-patch v4.9.0+incompatible
-	github.com/frankban/quicktest v1.10.0 // indirect
 	github.com/getsentry/sentry-go v0.7.0
-	github.com/ghodss/yaml v1.0.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/go-git/go-billy/v5 v5.0.0
 	github.com/go-git/go-git/v5 v5.1.0
 	github.com/go-openapi/runtime v0.19.22
 	github.com/go-openapi/spec v0.19.9 // indirect
 	github.com/go-openapi/strfmt v0.19.5
 	github.com/go-openapi/validate v0.19.11 // indirect
-	github.com/golang/snappy v0.0.2 // indirect
-	github.com/google/go-cmp v0.5.2
+	github.com/google/go-cmp v0.5.4
 	github.com/google/go-containerregistry v0.1.3
 	github.com/google/go-github/v32 v32.1.0
 	github.com/google/uuid v1.1.2
-	github.com/hashicorp/errwrap v1.1.0 // indirect
-	github.com/hashicorp/go-multierror v1.1.0 // indirect
 	github.com/hashicorp/go-retryablehttp v0.6.7
-	github.com/hashicorp/go-rootcerts v1.0.2 // indirect
-	github.com/hashicorp/vault/api v1.0.4
+	github.com/hashicorp/vault v1.7.1
+	github.com/hashicorp/vault/api v1.0.5-0.20210210214158-405eced08457
```

Comment:
`vault` and `vault/api` should probably identical ?

Reply:
When I set vault/api version to v1.7.1 and then try to build, I receive an error: `unknown revision api/v1.7.1`. Maybe you can suggest me how to fix this?

Reply:
Okay, I just checked the vault repository and they seem to use hierarchical git tagging. So `vault/api` has a separated versioning than `vault`. However `v1.0.5-0.20210210214158-405eced08457` is not clean. You can fix this by issuing `go get github.com/hashicorp/vault/api@latest` this will find the correct version for you to use.

Reply:
fixed